bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/NetatmoBridgeHandler.java

```diff
@@ -207,6 +207,8 @@ private String getApiScope() {
         if (configuration.readWelcome) {
             scopes.add("read_camera");
             scopes.add("access_camera");
+            scopes.add("read_presence");
+            scopes.add("access_presence");
```

Comment:
Thank you for your feedback.

I decided for the first option (renaming just label and description), because I think there is no use-case for enabling indoor camera access but not outdoor camera access or the other way round. I renamed it to generally "Access Camera API", that should it clarify for the user, it's downward compatible and nobody has to activate the Presence separately.

Reply:
Update: Hm, maybe it's better to separate it. It would cause that the user automatically grants more scopes when the binding is already installed, right? So it would be better when the user explicitly set the grants for the new scopes. I will separate it, please block the pull-request since the change is done.

Reply:
done. It is now separated, the user has to grant the access to the outdoor cameras explicitly.